pkg/controller/service/service_controller.go

```diff
@@ -136,6 +136,7 @@ func New(
 		cache.ResourceEventHandlerFuncs{
 			AddFunc: func(cur interface{}) {
 				svc, ok := cur.(*v1.Service)
+				// Check cleanup here can provide a remedy when controller failed to handle changes before it exiting (e.g. crashing, restart, etc.).
```

Comment:
nit: Could you wrap the comment into two lines?
```golang
// Check cleanup here can provide a remedy when controller failed to handle
// changes before it exiting (e.g. crashing, restart, etc.).
```

Reply:
Sure.